airbyte-integrations/connectors/source-mixpanel/source_mixpanel/spec.json

```diff
@@ -60,14 +61,14 @@
       "project_id": {
         "order": 1,
         "title": "Project ID",
-        "description": "Your project ID number. See the <a href=\"https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#project-id\">docs</a> for more information on how to obtain this.",
+        "description": "Your project ID number. See the <a href=\"https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#project-id\">docs</a> for more information on how to obtain this. Required if you are using a service account to authenticate.",
```

Comment:
Changed parameter to required

Reply:
Ah that looks like it will do it, I think - thanks! in that case we can probably remove the `Required if you are using a service account to authenticate.`